templates/conference.conf.xml

```diff
@@ -69,9 +69,17 @@
       <!-- File to play to acknowledge failure -->
       <!-- <param name="nack-sound" value="beeperr.wav"/> -->
       <!-- File to play to acknowledge muted -->
+      {% if bbb_freeswitch_muted_sound == true -%}
       <param name="muted-sound" value="conference/conf-muted.wav"/>
+      {% else -%}
+      <!-- <param name="muted-sound" value="conference/conf-muted.wav"/> -->
+      {% endif -%}
       <!-- File to play to acknowledge unmuted -->
+      {% if bbb_freeswitch_unmuted_sound == true -%}
       <param name="unmuted-sound" value="conference/conf-unmuted.wav"/>
+      {% else -%}
+      <!-- <param name="unmuted-sound" value="conference/conf-unmuted.wav"/> -->
+      {% endif -%}
```

Comment:
Are you sure this is the right indentation?
See:
https://svn.python.org/projects/external/Jinja-1.1/docs/build/escaping.html
#112

Reply:
Yes I am sure. I tested it several times by setting the vars true/false.

At first I didn't used the `-`, see https://github.com/ebbba-org/ansible-role-bigbluebutton/pull/115/commits/626b5de7489df86599b062cea86d50f0fd8ee15c but I think it looks better with the minus.